src/components/UpdatesFeed/index.test.js

```diff
@@ -0,0 +1,80 @@
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+
+import useRecentActivity from "queries/useRecentActivity";
+
+import UpdatesFeed from "../UpdatesFeed";
+
+import { HOME_TAB_KEYS } from "constants/home";
+
+jest.mock("queries/useRecentActivity");
+
+describe("UpdatesFeed tests", () => {
+	let mockData, setSelectedTab;
+
+	beforeEach(() => {
+		mockData = [
+			{
+				_id: "1",
+				content: "Update 1 content",
+				dateUpdated: "2022-09-13T23:03:02.878Z",
+				headline: "Update 1 headline",
+				documentUrl: "https://example.com/document1",
+				project: { name: "Project 1" },
+				pcp: { name: "PCP 1" },
+			},
+			{
+				_id: "2",
+				content: "Update 2 content",
+				dateUpdated: "2022-09-13T23:03:02.878Z",
+				headline: "Update 2 headline",
+				documentUrl: "https://example.com/document2",
+				project: { name: "Project 2" },
+				pcp: { name: "PCP 2" },
+			},
+		];
+		setSelectedTab = jest.fn();
+		useRecentActivity.mockReturnValue({ isError: false, isSuccess: true, data: mockData });
+	});
+
+	describe("UpdatesFeed header rendering", () => {
+		test("should render the feed header content", () => {
+			render(<UpdatesFeed setSelectedTab={setSelectedTab} />);
+
+			expect(screen.getByText("Updates")).toBeInTheDocument();
+			expect(screen.getByRole("button", { name: "Search all Updates >" })).toBeInTheDocument(
+				"Search all Updates button should be rendered",
```

Comment:
Please remove this text. Am I wrong? Have you seen this text rendered in the console? It seems redundant regardless.